.github/workflows/build-upstream-nightly.yml

```diff
@@ -17,3 +17,26 @@ jobs:
     secrets:
       SERVER_SSH_KEY: ${{ secrets.SERVER_SSH_KEY }}
       REMOTE_USER: ${{ secrets.REMOTE_USER }}
+  compute-matrix:
```

Comment:
This job key collides with the internal `compute-matrix` of the integrity check. You could rename to `compute-nightly-service-versions` to correct.

Reply:
Ahhh! That makes sense, thank you sempai